app/services/claims/claim/generate_csv.rb

```diff
@@ -1,7 +1,7 @@
 require "csv"
 
 class Claims::Claim::GenerateCSV < ApplicationService
-  HEADERS = %w[claim_reference urn school_name local_authority claim_amount establishment_type date_submitted].freeze
+  HEADERS = %w[claim_reference urn school_name local_authority claim_amount type_of_establishment establishment_type date_submitted claim_status].freeze
```

Comment:
@Nitemaeric can I just double check that this is correct?

There's already a column called `establishment_type`, and this PR adds a new column with a very similar name `type_of_establishment` but a different value.

| Header                  | Value                                |
|-------------------------|--------------------------------------|
| `type_of_establishment` | `claim.school.type_of_establishment` |
| `establishment_type`    | `claim.school.group`                 |

This is probably going to be a bit confusing unless ESFA are expecting both these columns.

Reply:
ESFA have specifically requested the `type_of_establishment` header as additional information.

As far as I'm aware, they're expecting an additional column to the existing pattern.

If it does cause any confusion moving forward, we can drop it easily enough :)